documentation/modules/security/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -3,9 +3,9 @@
 // assembly-security.adoc
 
 [id='installing-your-own-ca-certificates-{context}']
-= Installing your own CA certificates
+= Installing your own CA certificates for a Kafka cluster
 
-This procedure describes how to install your own CA certificates and private keys instead of using CA certificates and private keys generated by the Cluster Operator.
+This procedure describes how to install your own CA certificates and private keys at the cluster level instead of using CA certificates and private keys generated by the Cluster Operator.
```

Comment:
Again, I would probably leave out the cluster level. I do not think it is understandable what it means but people not really familiar with Strimzi.